cmd/server.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/metal-toolbox/conditionorc/internal/app"
+	"github.com/metal-toolbox/conditionorc/internal/model"
+	"github.com/metal-toolbox/conditionorc/internal/server"
+	"github.com/metal-toolbox/conditionorc/internal/store"
+	"github.com/spf13/cobra"
+	"go.hollow.sh/toolbox/events"
+)
+
+var (
+	shutdownTimeout = time.Second * 10
```

Comment:
small nit, but imo `10 * time.Second` reads nicer because it matches how we thing of numbers and units aka 10s.

Reply:
ah right thanks, not sure what I was thinking there :D